doc/format.rst

```diff
@@ -755,18 +755,25 @@ monospaced font; in reST, *double* backticks are for ``monospaced font``,
 whereas the behavior of single backticks is defined by the default role. This
 leads to the following style recommendations:
 
-- Module, class, function, method, and attribute names should render as
-  hyperlinks in monospaced font (e.g. :any:`numpy`); depending on project
-  settings, this may be accomplished simply be enclosing them in single
-  backticks. If the hyperlink does not render as intended, explicitly
-  include the appropriate role and/or namespace.
+- For example, module, class, function, method, and attribute names
+  should be enclosed in single backticks. On a case-by-case basis,
+  instances may also be included in single backticks, like :py:obj:`None`,
+  :py:obj:`True`, :py:obj:`False`.
+
+  These will typically render as hyperlinks in monospaced font (e.g.
+  :any:`numpy`) to the relevant object documentation; depending on project
+  settings. If the hyperlink does not render properly or point to the intended
```

Comment:
```suggestion
  :any:`numpy`) to the relevant object documentation.
  If the hyperlink does not render properly or point to the intended
```
Semicolon is no longer the appropriate punctuation, and I think this part can just be removed now.